modules/openapi-generator/src/main/resources/python/model_templates/classvars.mustache

```diff
@@ -130,7 +161,12 @@
 {{/mappedModels}}
         val = {
 {{#mappedModels}}
+{{#imports}}
```

Comment:
This may not work. It looks like it is looping over every import.
You probably want:
{{#imports}}
{{#-first}}
{{/-first}}
{{/imports}}